pandas/core/arrays/base.py

```diff
@@ -992,6 +995,35 @@ def _ndarray_values(self) -> np.ndarray:
         """
         return np.array(self)
 
+    def _accumulate(self, name, skipna=True, **kwargs):
+        """
+        Return an array result of performing the accumulation operation.
+
+        Parameters
+        ----------
+        name : str
+            Name of the function, supported values are:
+            { cummin, cummax, cumsum, cumprod }.
```

Comment:
Make this a list.

Can you put the signature of the callable? Do we know what this is?